lib/CLI.js

```diff
@@ -331,7 +331,7 @@ ${packageData.name} v${packageData.version} \
     if (!cliConfig.path) { cliConfig.path = []; }
     cliConfig.path.push(this.argv._[0]);
 
-    cliConfig.custom = this.custom;
+    cliConfig.custom = R.mergeDeepRight(this.custom, cliConfig.custom);
```

Comment:
Please, do you mind adding a value fallback and explanatory text?

```js
    // Merge "this.custom" which is an input of CLI constructior
    // (used for internal testing), and "cliConfig" which is a result
    // of merge upon "argv". Otherwise "custom" key from "dredd.yml"
    // is always overriden by "this.custom".
    cliConfig.custom = R.mergeDeepRight(this.custom, cliConfig.custom || {});
```

Otherwise it does seem to merge `custom` properties correctly. Nice job.

Reply:
I'll do it in few minutes.

Reply:
should i amend the previous commit?

Reply:
Sure, it's okay.